packages/terra-alert/src/Alert.module.scss

```diff
@@ -122,9 +122,11 @@
     box-shadow: var(--terra-alert-success-box-shadow, 2px 0 0 #78c346 inset);
   }
 
+  /* As per the W3C spec if the box-shadow does not have the color defined, 
+  it will use the prevailing color style */
```

Comment:
Could you move this comment to be inline with the box-shadow style.

Reply:
fixed here https://github.com/cerner/terra-core/pull/2407/commits/ffe3ff377f99567cc3e6d1d5c1be4eb714f01fe1.